### ISSUE TITLE
GrNet produces NaN entries in input tensor

### ISSUE THREAD

**Issue by SouvikBan** [master/tensorflow-riemopt#4]
Hi! First of all, really appreciate you guys taking the time to build a much required riemmannian geometry based package in tensorflow. It is proving to be quite useful for me. 
However, I recently ran the [GrNet code] (https://github.com/master/tensorflow-riemopt/tree/master/examples/grnet) with the AFEW dataset(the default dataset used in the code) on my machine and it seems at some point the input tensors get filled with NaN values. I tried tinkering with the learning rate and a few other usual things that could determine the cause of such NaN value in a dl model but it seems to be of no use. Any idea as to why this might be the case- is the code still been checked for bugs or am I missing something? Thanks in advance!

**Comment by ahariri13** [comment]
Hi @SouvikBan, were you able to get past this issue ? Any help would be appreciated :)

**Comment by SouvikBan** [comment]
Hi @ahariri13, unfortunately no. I was not able to get past this issue and since then have moved on to pytorch libraries like mcTorch and geoopt, although you will just have to code it yourself in those libraries.
